tests/README.md

```diff
@@ -0,0 +1 @@
+Wprowadzenie do testów z wykorzystaniem biblioteki pytest [Pytest Unit Testing Tutorial • How to test your Python code](https://www.youtube.com/watch?v=YbpKMIUjvK8)
```

Comment:
Lepiej na ten link założyć notatnik w tutorials. 
W tym pliku bardziej bym widział jak uruchomić wszystkie testy
Coś bardziej jak : 
Aby uruchomić wszystkie testy należy wykonać
```
pytest tests/* 
```

Reply:
Zmieniłem opis, a link z poradnikiem wrzuciłem do opisu zadania na ClickUp-ie.

Reply:
Mała szansa, że ktoś z was wróci po to do starego zadania.